### PR TITLE
Main Viewer Upgrade

### PR DESCRIPTION
**Pull request by ackagel**
Seeks to address most steps referenced in issue #2 .

## Completed steps
 - Delete figure/plot button added
 - Breakout figure windows
 - Contrast slider

## Remaining steps
 - Default colormap options

## Additional features added
 - Deleted cohort plots are automatically unchecked in the cohort tiff list

## Other notes
While custom colormaps are nice and could theoretically be implemented, it's not really a priority right now

### CODE REVIEW

**Comment by ackagel** [comment]
started back up on this again, could use some incremental reviews

File: main_viewer.py [hunk 1]
```diff
@@ -198,6 +206,16 @@ def add_plugin(self) -> None:
                 ui_name,
                 self.create_plugin_callback(ui_name))
 
+    # TODO: Write this function
```
Comment (alex-l-kong) [inline main_viewer.py]: Do we still need this TODO?

**Review by alex-l-kong — COMMENTED** [review]
Looking good!

File: main_viewer.py (hunk 1 above)
Comment (ackagel) [reply main_viewer.py]: noop

**Review by alex-l-kong — APPROVED** [review]
Looks good! I think some functions/classes could use some more documentation, but I'm not too worried about it for now because I don't think many people are going to be developing on top of this, at least for a while.

**Comment by ackagel** [comment]
thoughts on what functions/classes need documentation the most right now?  planning on adding in your feedback, merging this in, and then working on distributability

**Pull request merged**